### ISSUE TITLE
Result Selection Method in Stage Three

### ISSUE THREAD

**Issue by self-reward** [shaoshitong/EDC#1]
Dear Authors,

Thank you for your significant contributions to this field. I have a question regarding the results saved during stage three of your study. I noticed that you also saved the results labeled as "best". Could you please explain the rationale behind saving the results from the best-performing epoch?
 
I look forward to your clarification.

Best regards,

**Comment by shaoshitong** [comment]
Our code for this part comes from [RDED post-training](https://github.com/LINs-lab/RDED/blob/main/validation/main.py) and [SRe2l](https://github.com/VILA-Lab/SRe2L?tab=readme-ov-file) (i.e., wandb), we only use it for observation. The fact that this data is recorded does not mean that we report it in the paper using the best results in wandb.  We only report model performance at the end of training. In particular, the performance fluctuations are very small for post-training (stage 3), while on ImageNet-1k the best results usually are obtained at epoch=300. For Tiny-ImageNet and ImageNet-10 it is likely to be best at epoch=270/280.

**Comment by self-reward** [comment]
Thank you for your explanation. I attempted to replicate the CIFAR-10 results using the provided commands, but the performance was not as expected. Could this be due to differences in the hyperparameter settings?

**Comment by shaoshitong** [comment]
Hi As I mentioned in README.md, now we only ensure the performance on ImageNet-1k. The performance on CIFAR-10 cannot is not expected because we haven’t organized this part and the hyperparameter in the published code is not optimal.

**Comment by self-reward** [comment]
OK, Thank you for your great works again.
